web/packages/extension/src/utils.ts

```diff
@@ -131,8 +131,8 @@ if (typeof chrome !== "undefined") {
             chrome.tabs.create({ url: "/options.html" }, cb),
         );
     openPlayerPage = () =>
-        promisify((_cb: () => void) =>
-            chrome.tabs.create({ url: "/player.html" }),
+        promisify((cb: () => void) =>
+            chrome.tabs.create({ url: "/player.html" }, cb),
         );
 } else if (typeof browser !== "undefined") {
```

Comment:
Drive-by: This `if` probably should be first, because Firefox supports both `chrome` and `browser`.